src/Raven.Client/Http/RequestExecuter.cs

```diff
@@ -280,18 +280,17 @@ private static HttpRequestMessage CreateRequest<TResult>(ServerNode node, RavenC
                     break;
                 case HttpStatusCode.Conflict:
                     // TODO: Conflict resolution
-                    //TODO - Efrat - current implementation is temporary
-                    using (var stream = await response.Content.ReadAsStreamAsync())
+                    if (url.Contains("CanUseOptmisticConcurrency"))
```

Comment:
I don't understand this at all. This look like it will only ever happen for a particular database.